doc_source/aws-resource-cognito-userpoolclient.md

```diff
@@ -131,7 +131,7 @@ App callback URLs such as myapp://example are also supported\.
 *Update requires*: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
 
 `ExplicitAuthFlows`  <a name="cfn-cognito-userpoolclient-explicitauthflows"></a>
-The explicit authentication flows, which can be one of the following: `ADMIN_NO_SRP_AUTH`, `CUSTOM_AUTH_FLOW_ONLY`, or `USER_PASSWORD_AUTH`\.  
+The explicit authentication flows, which can be one of the following: `ALLOW_CUSTOM_AUTH`, `ADMIN_NO_SRP_AUTH`, `ALLOW_REFRESH_TOKEN_AUTH`, `ALLOW_USER_PASSWORD_AUTH`, `ALLOW_USER_SRP_AUTH`, `CUSTOM_AUTH_FLOW_ONLY`, `USER_PASSWORD_AUTH`, `ALLOW_ADMIN_USER_PASSWORD_AUTH`\.  
```

Comment:
You removed "or" before last option.

Reply:
Added missing "or" in commit 2dbc093f2ad66f9fb74e4d278449d111e9e87457